lib/realTimeClock/realTimeClock.h

```diff
@@ -0,0 +1,39 @@
+/*
+  realTimeClock.h - A c++ library to initialize the Teensy 4.1's real-time clock and related functions.
+  Created by Salvador Salcedo.
+*/
+
+#ifndef __REALTIMECLOCK__
+#define __REALTIMECLOCK__
+
+#include <TimeLib.h>
+#include <DS1307RTC.h>
+#include <Arduino.h>
+
+using namespace std;
+
+namespace RealTimeClock {
+
+    void init() {
+      if(timeStatus() != timeSet) 
+        setSyncProvider(RTC.get);
+    }
+
+    String getFileTime() {
+      String result = String(month()).concat("-");
+      result.concat(day());
+      result.concat("-");
```

Comment:
let's use `std::string` instead of Arduino's `String` we're trying to move in that direction.